cpp/include/raft/neighbors/detail/cagra/bitonic.hpp

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
Is it possible to use the bitonic sort in `cpp/include/raft/util/bitonic_sort.cuh ` here?